modules/caddyhttp/metrics.go

```diff
@@ -122,7 +122,7 @@ func (h *metricsInstrumentedHandler) ServeHTTP(w http.ResponseWriter, r *http.Re
 
 	// This is a _bit_ of a hack - it depends on the ShouldBufferFunc always
 	// being called when the headers are written.
-	// Effectively the same behaviour as promhttp.InstrumentHandlerTimeToWriteHeader.
+	// Effectively the same behavior as promhttp.InstrumentHandlerTimeToWriteHeader.
```

Comment:
🙄 lol

Reply:
I just realized - the problem here is that `locale` is explicitly set to `US` - but it's not an either/or setting! After looking at the docs I'm realizing that if I simply unset it, it'll allow both varieties 😬 🇨🇦🇺🇸

Reply:
amazing 😁